types/recharts/index.d.ts

```diff
@@ -271,14 +272,14 @@ export interface LineProps {
 	dot?: boolean | any | React.ReactElement<any> | RechartsFunction;
 	activeDot?: boolean | any | React.ReactElement<any> | RechartsFunction;
 	label?: boolean | any | React.ReactElement<any> | RechartsFunction;
-	points: any[];
+	points?: any[];
```

Comment:
This is an interesting one. The Recharts docs for 0.22 do not say it's optional, yet all the examples don't list them. You think we should add a comment to clarify on this?